### ISSUE TITLE
Unexpected behaviour or zonal_sum

### ISSUE THREAD

**Issue by OliverSchmitz** [computationalgeography/lue#594]
I do `tmp = lfr.zonal_sum(clone, clone)`, with clone read from a GeoTIFF and of type GDT_Byte. The result map contains incorrect values as the sum in my case exceeds the value range of the result map. 
I'd prefer to get an error message instead

**Comment by kordejong** [comment]
What should happen, given the code that is in place (and apparently contains a bug), is that out-of-range values end up as no-data in the result. Would that work for you?
We could also (optionally) print warnings when the number of no-data values increase during an operation. This is also something that you can do yourself in the script, using a combination of `is_valid` and `sum` for example (not sure if those names are correct).

**Comment by OliverSchmitz** [comment]
Yes, I think no-data would be ok, at least an indication that something is not working as desired...
